app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/rootnav/RootNavViewModel.kt

```diff
@@ -151,6 +151,10 @@ class RootNavViewModel @Inject constructor(
                         )
                     }
 
+                    SpecialCircumstance.AccountSecurityShortcut -> {
+                        RootNavState.VaultUnlocked(activeUserId = userState.activeAccount.userId)
+                    }
```

Comment:
Can we group it with the other shortcuts?

```kotlin
SpecialCircumstance.AccountSecurityShortcut,
SpecialCircumstance.GeneratorShortcut,
pecialCircumstance.VaultShortcut,
null,
    -> RootNavState.VaultUnlocked(activeUserId = userState.activeAccount.userId)
```

Reply:
I like it :) 
39aafa568ce751c0316af1cba73a40983434d1eb